README.md

```diff
@@ -107,6 +107,7 @@ you'll need to do the following:
     - Postsubmit jobs run after merging code
     - Periodic job run on a timed basis
     - You can find more prowjob definitions at [how-to-add-new-jobs](prow#how-to-add-new-jobs)
+  - Try to utilize [podutils](prow/pod-utilities.md#how-to-configure) to create modern prowjobs!
```

Comment:
suggestion: "Please utilize the [podutils] ..."

Reply:
👍